bigchaindb/commands/utils.py

```diff
@@ -20,6 +20,10 @@ def start_rethinkdb():
 
     for line in proc.stdout:
         if line.startswith('Server ready'):
+            # FIXME: seems like tables are not ready when the server is ready,
```

Comment:
OK, tried again, seems to work if the table has been created (and it's not yet ready) :+1: